com.unity.shadergraph/Editor/ShaderGraphProjectSettings.cs

```diff
@@ -0,0 +1,91 @@
+using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph
+{
+    [FilePath("ProjectSettings/ShaderGraphProjectSettings.asset", FilePathAttribute.Location.ProjectFolder)]
+    internal class ShaderGraphProjectSettings : ScriptableSingleton<ShaderGraphProjectSettings>
+    {
+        [SerializeField]
+        internal int customInterpolatorErrorThreshold = 32;
+        [SerializeField]
+        internal int customInterpolatorWarningThreshold = 16;
+        internal SerializedObject GetSerializedObject() { return new SerializedObject(this); }
+        internal void Save() { Save(true); }
+        private void OnDisable() { Save(); }
+    }
+
+    class ShaderGraphProjectSettingsProvider : SettingsProvider
+    {
+        private static string kCustomInterpolatorHelpBox = "These options are used to help ShaderGraph users maintain known compatibilities with target platform(s) when using Custom Interpolators.";
```

Comment:
Proposed rewrite (also mentioned in another "omnibus" comment on this file):
Unity uses these options to help ShaderGraph users maintain known compatibilities with target platform(s) when using Custom Interpolators

Reply:
Change is already in.

Reply:
Great! (Found the relevant commit).